charts/incubator/blocky/values.yaml

```diff
@@ -0,0 +1,249 @@
+# Primarily designed for inclusion in the TrueNAS SCALE GUI
+conditional:
+  rewrite:
+  # - in: something.com
+  #   out: somethingelse.com
+  mapping:
+  # - domain: something.com
+  #   dnsserver: 192.168.178.1
+
+# -- set blocking settings using Lists
+# Primarily designed for inclusion in the TrueNAS SCALE GUI
+blocking:
+  # -- Sets the blocktype
+  blockType: nxDomain
+  # -- Sets the block ttl
+  blockTTL: 6h
+  # -- Sets the block refreshPeriod
+  refreshPeriod: 4h
+  # -- Sets the block download timeout
+  downloadTimeout: 60s
+  # -- Sets the block download attempt count
+  downloadAttempts: 3
+  # -- Sets the block download cooldown
+  downloadCooldown: 2s
+  # -- Set to fail start of lists cannot be downloaded
+  failStartOnListError: false
+  # -- Sets how many list-groups can be processed at the same time
+  processingConcurrency: 4
+  # --  Add blocky whitelists
+  whitelist: []
+  #  - name: ads
+  #    lists:
+  #      - https://someurl.com/list.txt
+  #      - /somefile.txt
+
+  # -- Blocky blacklists
+  blacklist: []
+  # - name: ads
+  #   lists:
+  #     - https://someurl.com/list.txt
+  #     - /somefile.txt
+
+  # -- Blocky clientGroupsBlock
+  clientGroupsBlock: []
+  # - name: default
+  #   groups:
+  #     - ads
+
+k8sgateway:
+  enabled: true
+  # -- TTL for non-apex responses (in seconds)
+  ttl: 300
+
+  # -- Limit what kind of resources to watch, e.g. watchedResources: ["Ingress"]
+  watchedResources: []
+
+  # -- Service name of a secondary DNS server (should be `serviceName.namespace`)
+  secondary: ""
+
+  # -- Override the default `serviceName.namespace` domain apex
+  apex: ""
+
+  # -- list of processed domains
+  domains: []
+  # -- Delegated domain
+  # - domain: "example.com"
+  #   # -- Optional configuration option for DNS01 challenge that will redirect all acme
+  #   # challenge requests to external cloud domain (e.g. managed by cert-manager)
+  #   # See: https://cert-manager.io/docs/configuration/acme/dns01/
+  #   dnsChallenge:
+  #     enabled: false
+  #     domain: dns01.clouddns.com
+
+  forward:
+    enabled: false
+    primary: tls://1.1.1.1
+    secondary: tls://1.0.0.1
+    options:
+      - name: tls_servername
+        value: cloudflare-dns.com
+
+serviceAccount:
+  main:
+    # -- Specifies whether a service account should be created
+    enabled: true
+
+# -- Create a ClusterRole and ClusterRoleBinding
+# @default -- See below
+rbac:
+  main:
+    # -- Enables or disables the ClusterRole and ClusterRoleBinding
+    enabled: true
+
+    # -- Set Rules on the ClusterRole
+    rules:
+      - apiGroups:
+          - ""
+        resources:
+          - services
+          - namespaces
+        verbs:
+          - list
+          - watch
+      - apiGroups:
+          - extensions
+          - networking.k8s.io
+        resources:
+          - ingresses
+        verbs:
+          - list
+          - watch
+
+redis:
+  enabled: true
+  existingSecret: "rediscreds"
+
+portal:
+  enabled: true
```

Comment:
please move this above redis, we want dependencies on the bottom, to highlight they are technically subcharts